### ISSUE TITLE
Cannot find 'Channels' property when starting bot.

### ISSUE THREAD

**Issue by QSmally** [discordjs/discord.js#3169]
**Please describe the problem you are having in as much detail as possible:**
After I did some coding of my bot, the bot worked beautifully and such. Until I restarted it (without editing anything at all, which worked before), and then on startup, it crashed saying that it couldn't define a channel within the Discord library.

**Include a reproducible code sample here, if possible:**
```prolog
/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/structures/Guild.js:235
          this.channels.get(voiceState.channel_id).members.set(member.user.id, member);
                                                  ^

TypeError: Cannot read property 'members' of undefined
    at Guild.setup (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/structures/Guild.js:235:51)
    at GuildCreateHandler.handle (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/client/websocket/packets/handlers/GuildCreate.js:12:15)
    at WebSocketPacketManager.handle (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
    at WebSocketConnection.onPacket (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:333:35)
    at WebSocketConnection.onMessage (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:296:17)
    at WebSocket.onMessage (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:180:13)
    at Receiver._receiver.onmessage (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/ws/lib/websocket.js:137:47)
    at Receiver.dataMessage (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/ws/lib/receiver.js:397:14)
    at Receiver.getData (/Users/joey/Desktop/dev/QBot-Source/Main/QBot/node_modules/ws/lib/receiver.js:347:12)
```

As it seems like it, the package is getting invalid data of some sort. Because the user ID isn't a valid ID, such as the channel (why it's throwing the error, the channel can't resolve due to the channel ID being not valid).

For now, I will add a line of code to return when the channel isn't valid for a temporary fix until somebody has an answer, fix, or redirect. Help is much appreciated.

Oh also, when logging in on a different bot user token, the error does not apply. Only if I run it on this specific token **or** running the bot on another account in the same guilds. Both the `user_id` and `channel_id` from the raw data it gets are invalid as of the client. (Found by doing `Client.channels/users.get(<id>);`)

**Further details:**
- discord.js version: 11.4.2
- Node.js version: 6.8.0
- Operating system: MacOS Mojave version 10.14.3 (18D109)
- Priority this issue should have – please be realistic and elaborate if possible:

- [ ] I have also tested the issue on latest master, commit hash:

**Comment by Monbrey** [comment]
I think we'd need to see a bit more of your code in order to determine the cause of this issue, where this is being run etc. I encourage you to discuss it in the [Discord.js Official](https://discord.gg/bRCvFy9) Discord server.

**Comment by izexi** [comment]
This has been fixed on the 11.4-dev branch, specifically on this commit https://github.com/discordjs/discord.js/commit/73aaab51061b21cc21f0102703ba203e155d359d.

You can update to 11.4-dev by doing `npm i discordjs/discord.js#11.4-dev`. 11.4-dev consists of bug fixes and backports from the master (12.0.0-dev) branch so there aren't any breaking changes from stable (11.4.2) to 11.4-dev.
